contracts/gateway/Anchor.sol

```diff
@@ -156,18 +158,18 @@ contract SafeCore is StateRootInterface, Organized {
     }
 
     /**
-     *  @notice External function commitStateRoot.
+     *  @notice External function anchorStateRoot.
      *
-     *  @dev commitStateRoot Called from game process.
-     *       Commit new state root for a block height.
+     *  @dev anchorStateRoot Called from game process.
+     *       Anchor new state root for a block height.
```

Comment:
Please clean up these doc comments. they don't really fit. The `@notice` doesn't help and the `@dev` is also somewhat redundant. Maybe remove the `@dev` and make the `@notice`:

> Anchor a new state root for a given block height.